.npmignore

```diff
@@ -13,21 +13,3 @@
 !dist/**/*.js
 !dist/**/*.d.ts
 !dist/**/*.map
-
-# Include this as a code sample
-!Makefile
-
-# Include the example app
```

Comment:
Why are you removing this? How are people meant to access it?

Reply:
this should not be part of the bundled npm package. ideally, examples live outside of the library and `npm install` should not also get the examples into the `node_modules`. we risk adding a lot of not needed code to the packed code as we add more examples